src/main/java/mate/academy/internetshop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,151 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import mate.academy.internetshop.dao.OrderDao;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import mate.academy.internetshop.model.Order;
+import org.apache.log4j.Logger;
+
+@Dao
+public class OrderDaoJdbcImpl extends AbstractDao<Order> implements OrderDao {
+    private static Logger logger = Logger.getLogger(OrderDaoJdbcImpl.class);
+
+    public OrderDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    public List<Item> addItemsInDb(Order order) {
+        List<Item> items = order.getItems();
+        for (Item item : items) {
+            String query = "INSERT INTO orders_items (order_id, item_id) VALUES (?, ?);";
+            try (PreparedStatement statement = connection.prepareStatement(query)) {
+                statement.setLong(1, order.getId());
+                statement.setLong(2, item.getId());
+                statement.executeUpdate();
+            } catch (SQLException e) {
+                logger.error("Can't add items for order with id = " + order.getId(), e);
+            }
+        }
+        return items;
+    }
+
+    public List<Item> getItemsFromDb(Long orderId) {
+        List<Item> items = new ArrayList<>();
+        String query = "SELECT * FROM items INNER JOIN orders_items"
+                + " USING (item_id) WHERE order_id = ?;";
+        try (PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, orderId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                Long itemId = resultSet.getLong("item_id");
+                String name = resultSet.getString("name");
+                Double price = resultSet.getDouble("price");
+                Item item = new Item(name, price);
+                item.setId(itemId);
+                items.add(item);
+            }
+        } catch (SQLException e) {
+            logger.error("Can't get items for order with id = " + orderId, e);
+        }
+        return items;
+    }
+
+    public void deleteItemFromDB(Long orderId) {
```

Comment:
```suggestion
    private void deleteItemFromDB(Long orderId) {
```
@TatianaPirch can this method be private?